src/Public.Api/Infrastructure/Redis/RedisStoreKeyGenerator.cs

```diff
@@ -3,6 +3,7 @@ namespace Public.Api.Infrastructure.Redis
     using System.Text.RegularExpressions;
     using System.Threading.Tasks;
     using Common.Infrastructure;
+    using Common.Infrastructure.Extensions;
```

Comment:
Random new using statement can probably be removed?

Reply:
Seems to come from this commit [`34c6752` (#76)](https://github.com/Informatievlaanderen/public-api/pull/76/commits/34c6752a20e192bf80988372de12286af626e42f)

Reply:
Extensions are moved to a folder to reduce clutter.
`using Common.Infrastructure;` could be removed though ;)

Reply:
Maybe make a `ContentFormat.Set` extension method which sets it again? :)

Reply:
Yes! :D